examples/main/src/layer-examples.js

```diff
@@ -88,6 +89,31 @@ const GeoJsonLayerExample = {
   }
 };
 
+const GeoJsonLayer64Example = {
+  layer: GeoJsonLayer64,
+  props: {
```

Comment:
Just a thought, we could update this so that each example had a

   `layer`: 
   `layer64`: 

To avoid a never ending list. But would take some UI work. Maybe wait until we fully integrate the two variants.